src/Spice86/ViewModels/CpuViewModel.cs

```diff
@@ -25,13 +25,16 @@ public partial class CpuViewModel : ViewModelBase, IEmulatorObjectViewModel {
     [ObservableProperty]
     private CpuFlagsInfo _flags = new();
 
+    [ObservableProperty] private bool _isCfgCpuEnabled;
+
     public CpuViewModel(State state, IMemory memory, IPauseHandler pauseHandler, IUIDispatcher uiDispatcher) {
         _cpuState = state;
         _memory = memory;
         pauseHandler.Paused += () => uiDispatcher.Post(() => _isPaused = true);
         _isPaused = pauseHandler.IsPaused;
         pauseHandler.Resumed += () => uiDispatcher.Post(() => _isPaused = false);
         DispatcherTimerStarter.StartNewDispatcherTimer(TimeSpan.FromMilliseconds(400), DispatcherPriority.Background, UpdateValues);
+        IsCfgCpuEnabled = false;
```

Comment:
I would not set this to enabled by default, but rather pass the Configuration object. It is the single source of truth to know if the CfgCpu is enabled.